assets/src/domain/eventEditor/ui/ticketAssignmentsManager/components/TicketAssignmentsManagerModal/index.tsx

```diff
@@ -7,27 +7,25 @@ import CloseModalButton from './buttons/CloseModal';
 import TicketAssignmentsManager from '../TicketAssignmentsManager';
 import useCancelButtonProps from './buttons/useCancelButtonProps';
 import useSubmitButtonProps from './buttons/useSubmitButtonProps';
-import { useTAMContext } from '../../context';
+import { useDataState } from '../../data';
+import { WithContextProps } from '../../context/types';
 
 import '../styles.scss';
 
-const TicketAssignmentsManagerModal: React.FC = () => {
-	const {
-		dataState: { hasOrphanEntities },
-		onCloseModal,
-		title,
-	} = useTAMContext();
```

Comment:
Using context here caused re-renders.

Reply:
would be nice to document this conclusion somewhere, along with that comment you posted yesterday: https://github.com/facebook/react/issues/15156#issuecomment-474590693